examples/release_highlights/plot_release_highlights_0_22_0.py

```diff
@@ -0,0 +1,119 @@
+"""
+========================================
+Release Highlights for scikit-learn 0.22
+========================================
+
+We are pleased to announce the release of scikit-learn 0.22, which comes
+with many bug fixes and new features! We detail below a few of the major
+features of this release. For an exhaustive list of all the changes, please
+refer to the :ref:`release notes <changes_0_22>`.
+
+To install the latest version (with pip)::
+
+    pip install -U scikit-learn --upgrade
+
+or with conda::
+
+    conda install scikit-learn
+"""
+
+##############################################################################
+# Permutation-based feature importance
+# ------------------------------------
+#
+# The :func:`~sklearn.inspection.permutation_importance` can be used to get an
+# estimate of the importance of each feature, for any fitted estimator:
+
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.datasets import make_classification
+from sklearn.inspection import permutation_importance
+import matplotlib.pyplot as plt
+
+X, y = make_classification(random_state=0, n_features=5, n_informative=3)
+rf = RandomForestClassifier(random_state=0).fit(X, y)
+result = permutation_importance(rf, X, y, n_repeats=10, random_state=0,
+                                n_jobs=-1)
+
+fig, ax = plt.subplots()
+sorted_idx = result.importances_mean.argsort()
+ax.boxplot(result.importances[sorted_idx].T,
+           vert=False, labels=range(X.shape[1]))
+ax.set_title("Permutation Importance of each feature")
+ax.set_ylabel("Features")
+fig.tight_layout()
+plt.show()
+
+##############################################################################
+# Native support for missing values for gradient boosting
+# -------------------------------------------------------
+#
+# The :class:`~sklearn.ensemble.HistGradientBoostingClassifier`
+# and :class:`~sklearn.ensemble.HistGradientBoostingRegressor` now have native
+# support for missing values (NaNs). This means that there is no need for
+# imputing data when training or predicting.
+
+from sklearn.experimental import enable_hist_gradient_boosting  # noqa
+from sklearn.ensemble import HistGradientBoostingClassifier
+import numpy as np
+
+X = np.array([0, 1, 2, np.nan]).reshape(-1, 1)
```

Comment:
use a real-word dataset instead?

Reply:
I'd rather not, here it's clear that we deal with nans (and that the prediction is correct)